docs/contribute.html

```diff
@@ -0,0 +1,113 @@
+        </button>
+        <div class="collapse navbar-collapse" id="navbarResponsive">
+          <ul class="navbar-nav ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="about.html">About</a>
+            </li>
+            <li class="nav-item active">
+              <a class="nav-link" href="contribute.html">Contribute
+                <span class="sr-only">(current)</span>
+              </a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="download.html">Download</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    
+	<!-- Text Right, Image Left -->
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6">
+            <div class="p-5">
+              <img class="img-fluid" src="images/code.jpg" alt="Code Image">
+            </div>
+          </div>
+          <div class="col-md-6">
+            <div class="p-5">
+              <h2 class="display-4">Point of Contact</h2>
+              <p>You can view this project on Git-Hub, click <a href="https://github.com/rsanchez-wsu/RaiderPlanner" target="_blank">HERE</a> to open the Git-Hub project page in a new window.</p> 
+			  <p>You may also contact the lead of our project, Mr. Roberto Sanchez at <a href="mailto:roberto.sanchez@wright.edu">roberto.sanchez@wright.edu</a>.</p>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+
+	<!-- Text Left, Image Right -->
+    <section>
+      <div class="container">
+        <div class="row align-items-center">
+          <div class="col-md-6 order-2">
+            <div class="p-5">
+			  <!-- Image taken from http://www.pageinc.org/?06 -->
+              <img class="img-fluid rounded-circle" src="images/legal.jpg" alt="Legal Img">
+            </div>
+          </div>
+          <div class="col-md-6 order-1">
+            <div class="p-5">
+              <h2 class="display-4">Legal Terms</h2>
+              <p>Applicable to Original files (Alienturnedhuman/PearPlanner):
+			  <br>You may</p>
+			  <ul>
+				<li>keep copies of the files</li>
+				<li>include them for reference reasons in future releases</li>
+				<li>refer to them in your documentation (providing you correctly attribute and give credit)</li>
+			  </ul>
+			  <p>You may not:</p>
+			  <ul>
+				<li>modify them in anyway</li>
+				<li>attribute them to anyone other than the original authors</li>
+				<li>use them as your own work in any documentation you create</li>
+			  </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+	
+    <!-- Footer -->
+    <footer class="py-3 bg-dark">
+      <div class="container">
+        <p class="m-0 text-center text-white">Copyright &copy; RaiderPlanner 2017</p>
```

Comment:
RaiderPlanner is not an organization, so this should rather say "RaiderPlanner Contributors" and then link to the AUTHORS file in the project repository. I have just now added the file.